pandas_sphinx_theme/footer.html

```diff
@@ -11,22 +11,50 @@
 
 <script src="https://sidecar.gitter.im/dist/sidecar.v1.js" async defer></script>
 
-<footer class="footer">
-  <div class="container">
-    <p>
-      {%- if show_copyright %}
-        {%- if hasdoc('copyright') %}
-          {% trans path=pathto('copyright'), copyright=copyright|e %}&copy; <a href="{{ path }}">Copyright</a> {{ copyright }}.{% endtrans %}<br/>
-        {%- else %}
-          {% trans copyright=copyright|safe %}&copy; Copyright {{ copyright }}.{% endtrans %}<br/>
-        {%- endif %}
-      {%- endif %}
-      {%- if last_updated %}
-        {% trans last_updated=last_updated|e %}Last updated on {{ last_updated }}.{% endtrans %}<br/>
-      {%- endif %}
-      {%- if show_sphinx %}
-        {% trans sphinx_version=sphinx_version|e %}Created using <a href="http://sphinx-doc.org/">Sphinx</a> {{ sphinx_version }}.{% endtrans %}<br/>
-      {%- endif %}
-    </p>
-  </div>
-</footer>
\ No newline at end of file
+
+
+
+<div class="footer">
+    <div class="container pb-3">
+        <div class="row justify-content-center pb-3">
+            <a href="https://opencollective.com/{{theme_open_collective}}/donate" target="_blank">
+              <img src="https://www.spyder-ide.org/static/images/collective.svg" class="center donate" />
+            </a>
+        </div>
+        <div class="row justify-content-center">
+            <div class="text-center pb-3">
+                 Connect with us on: 
+                  {% if theme_github_url %}
+                  <a href="{{theme_github_url}}">Github</a>
+                  {% endif %}
+                  {% if theme_twitter_url %}
+                  <span class="pipe-red">|</span> <a href="{{theme_twitter_url}}">Twitter</a>
+                  {% endif %}
+                  {% if theme_facebook_url %}
+                  <span class="pipe-red">|</span> <a href="{{theme_facebook_url}}">Facebook</a>
+                  {% endif %}
+                  {% if theme_google_groups_url %}
+                  <span class="pipe-red">|</span> <a href="{{theme_google_groups_url}}">Google Groups</a>
+                  {% endif %}
+            </div>
+        </div>
+        <div class="row justify-content-center">
+              <small>
+                <div class="text-center">
+                  {% if show_copyright %}&copy;{{ copyright }}{% endif %}
+                </div>
```

Comment:
```suggestion
                {% if show_copyright %}
                <div class="text-center">
                  &copy;{{ copyright }}
                </div>
                {% endif %}
```

Shouldn't this be around the whole block?